common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1238,7 +1238,10 @@ def submit_problem(self, data, override_time=False):
             # without a stack trace
             else:
                 # Translators: {msg} will be replaced with a problem's error message.
-                msg = _(u"Error: {msg}").format(msg=inst.message)
+                if type(inst) == StudentInputError:
```

Comment:
I don't think we should add this conditional code-- either we should never say "Error" (which means that we have to make sure all messages start with a capitol letter and are clear enough that it is obvious they are an error condition), or we should always start with "Error".

Interestingly we aren't display this message with the Error notification, but instead with the warning one.